tools/dds/dds-adapter/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ target_include_directories( ${PROJECT_NAME} PRIVATE
 target_link_libraries( ${PROJECT_NAME} PRIVATE realdds realsense2 )
 set_target_properties (${PROJECT_NAME} PROPERTIES
     FOLDER Tools/dds
-    CXX_STANDARD 11
+    CXX_STANDARD 14
```

Comment:
Can we inherit this from a global setting that applies to SDK and all tools?

Reply:
There isn't one currently -- the main SDK is 14 but the tools and examples were all left at 11.